testdata/array_body/api/api.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+// PostPosts godoc
+// @Summary Add a lot of posts to somewhere
+// @Description Post lot of posts to somewhere
+// @Accept  json
+// @Produce  json
+// @Param posts body []web.Post true "This is a lots of posts"
```

Comment:
Why can't it be 
`// @Param posts body web.PostCollection true "This is a lots of posts"
`
and web.PostCollection defined as ?
`type PostCollection []Post
`